src/elements/content-uploader/UploadsManager.scss

```diff
@@ -2,9 +2,14 @@
 
 .be.bcu-uploads-manager-container {
     position: fixed;
+    right: 56px;
     bottom: 0;
-    width: 100%;
+    left: 288px;
+    width: unset;
     max-height: 0;
+    border-top-left-radius: 12px;
```

Comment:
Can these also use a variable as I can imagine that `border-radius` should be consistent ?

Reply:
The radius is only applied to this specific case - it won't be used anywhere else